cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -96,12 +98,32 @@ func startMizarPodController(ctx *ControllerContext, grpcHost string, grpcAdapto
 	controllerName := "mizar-pod-controller"
 	klog.V(2).Infof("Starting %v", controllerName)
 
-	go controllers.NewMizarPodController(
-		ctx.InformerFactory.Core().V1().Pods(),
-		ctx.ClientBuilder.ClientOrDie(controllerName),
-		grpcHost,
-		grpcAdaptor,
-	).Run(mizarPodControllerWorkerCount, ctx.Stop)
+	podKubeconfigs := ctx.ClientBuilder.ConfigOrDie(controllerName)
+	for _, podKubeconfig := range podKubeconfigs.GetAllConfigs() {
+		podKubeconfig.QPS *= 5
+		podKubeconfig.Burst *= 10
+	}
+
+	crConfigs := *podKubeconfigs
+	for _, cfg := range crConfigs.GetAllConfigs() {
+		cfg.ContentType = "application/json"
+		cfg.AcceptContentTypes = "application/json"
+	}
+	networkClient := arktos.NewForConfigOrDie(&crConfigs)
+	informerFactory := externalversions.NewSharedInformerFactory(networkClient, 10*time.Minute)
```

Comment:
The last argument is default resync period, i.e., how frequent the entire object storage need to be rescan and revalidate. Suggest change it to 0, meaning no resync.

Reply:
made the change.

Reply:
The changes works.